spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMerge.java

```diff
@@ -1186,6 +1186,32 @@ public void testMergeAlignsUpdateAndInsertActions() {
         sql("SELECT * FROM %s ORDER BY id", tableName));
   }
 
+  @Test
+  public void testMergeMixedCaseAlignsUpdateAndInsertActions() {
+    createAndInitTable("id INT, a INT, b STRING", "{ \"id\": 1, \"a\": 2, \"b\": \"str\" }");
+    createOrReplaceView(
+        "source",
+        "{ \"id\": 1, \"c1\": -2, \"c2\": \"new_str_1\" }\n" +
+            "{ \"id\": 2, \"c1\": -20, \"c2\": \"new_str_2\" }");
```

Comment:
nit: the rest of this file aligns the json records slightly differently (no extra indentation for 2nd record)

Reply:
ugh it was the intellij helper annotations, they made it look aligned :)

Reply:
Hahaha that happens to me all the time. So annoying.